src/views/SingleEventView.vue

```diff
@@ -191,9 +191,22 @@ export default {
       // eslint-disable-next-line no-alert
       alert(`Once created, link create-announcement component here for ${payload.event.title}.`);
     },
-    viewRSVP(event) {
-      // eslint-disable-next-line no-alert
-      alert(`${event.title} RSVP To be Implemented!`);
+    async viewRSVP(event) {
+      const resp = await api.getEventRSVP(this.eventId);
+      const isErr = resp && resp.response && resp.response.status !== 200;
+      if (!isErr) {
```

Comment:
This will pass if resp or resp.response is null, I think what you want is
`const isOkay = resp && resp.response && resp.response.status === 200;`

Reply:
Good catch, I think this fix should take care of `null` responses,  `undefined`  response, and responses that error with a non-200 status